data/json/monsters/mutant_mammal.json

```diff
@@ -364,5 +445,160 @@
       "SWARMS",
       "WARM"
     ]
+  },
+  {
+    "id": "mon_bigdog_greatestswiss",
+    "type": "MONSTER",
+    "name": { "str_sp": "Greatest Swiss" },
```

Comment:
Too confusing -  go with something descriptive instead. This is also a good time to set down a straightforward naming scheme for mutant dogs, especially if you plan on adding more, something like ` <adjective> <breed name>`. `enormous Greater Swiss` would work in a pinch.